apps/bare-expo/ios/Podfile.lock

```diff
@@ -244,6 +244,8 @@ PODS:
     - ExpoModulesCore
   - ExpoTrackingTransparency (3.2.0):
     - ExpoModulesCore
+  - ExpoVideo (0.1.0):
```

Comment:
Interesting that this isn't on `main` already 🤔

Reply:
Yeah, there's a few things not on main yet. `expo-head` is another. There was a lot of publishing last week.